types/tslint.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "dtslint/dtslint.json",
+  "rules": {
+    "semicolon": false,
+    "whitespace": false,
+    "trim-file": false,
+    "no-consecutive-blank-lines": false
```

Comment:
are the last two rule disables needed?
I'm not spotting what would trigger them.

```suggestion
    "whitespace": false
```

Reply:
This is related to the code style, my local editor will automatically add a blank line at the end of the file by default. If `no-consecutive-blank-lines` is not closed, my local will always be in a failed test state.

Maybe the project needs an [.editorconfig](https://github.com/mili-project-manager/mili/blob/master/.editorconfig) to unified code style. or eslint?

Reply:
We have an editorconfig: https://github.com/remarkjs/remark-autolink-headings/blob/main/.editorconfig. And pretty strict formatting (xo, which wraps eslint, and prettier).

Reply:
https://github.com/remarkjs/remark-autolink-headings/blob/main/.editorconfig#L8

```
 "no-consecutive-blank-lines": false
```

should not remove?

Reply:
I don’t think this has anything to do with that editorconfig setting: https://editorconfig.org

Reply:
I got it wrong